magenta/models/piano_genie/BUILD

```diff
@@ -66,6 +66,9 @@ py_binary(
     srcs = [
         "eval.py",
     ],
+		visibility = [
```

Comment:
nit: tabbing is inconsistent with the rest of the file

Reply:
Oops, not sure how that happened.  Does it look right now?